custom_components/simple_wyze_vac/vacuum.py

```diff
@@ -402,6 +408,29 @@ async def get_last_map(self):
         except:
             _LOGGER.warn("Failed to grab latest map image. Try again later.")
 
+    async def set_current_map(self, target_map=None):
```

Comment:
I think this should be mandatory, otherwise there's no point in calling the function with `None` since `target_map_id` will also be `None`  which then just throws a warning
```suggestion
    async def set_current_map(self, target_map):
```